4/alpine/Dockerfile

```diff
@@ -58,11 +58,10 @@ RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
   && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-  && mkdir -p /opt/yarn \
+  && mkdir -p /opt \
```

Comment:
Add `mkdir -p /opt`  to all the templates and image files, not just apline. As a general rule we try to make sure there is consistency across all variants and versions.

Reply:
fixed

Reply:
?

Reply:
ah no you said all images sry, will do

Reply:
done

Reply:
You missed the slim variant :)

Reply:
I checked if you were paying attention 😁

Reply:
:)